addons/sales_team/security/ir.model.access.csv

```diff
@@ -4,7 +4,7 @@ access_crm_team_user,crm.team.user,model_crm_team,sales_team.group_sale_salesman
 access_crm_team_manager,crm.team.manager,model_crm_team,sales_team.group_sale_manager,1,1,1,1
 access_crm_team_member_all,access.crm.team.member.all,model_crm_team_member,,0,0,0,0
 access_crm_team_member_user,access.crm.team.member.user,model_crm_team_member,base.group_user,1,0,0,0
-access_crm_team_member_salesman,access.crm.team.member.salesman,model_crm_team_member,sales_team.group_sale_salesman,1,1,1,1
+access_crm_team_member_salesman,access.crm.team.member.salesman,model_crm_team_member,sales_team.group_sale_manager,1,1,1,1
```

Comment:
Shouldn't you rename the rest of the line as well or is it annoying for migration reasons?